integration_tests/test_domestic_migration.py

```diff
@@ -239,32 +252,33 @@ def test_addresses_during_moves(simulants_on_adjacent_timesteps, unit_id_col, ad
         after_units = after_units.reindex(total_index)
         mask_moved_units = before_units[address_id_col] != after_units[address_id_col]
 
-        # check that the number of moved units that have the same details is very low
+        # Check that the number of moved units that have the same details is very low
         # NOTE: we cannot assert that all moved units have different details
-        # because they are free to move within the same state or even puma
+        # because they are free to move within the same state or even PUMA
         assert (
             (
-                before_units.loc[mask_moved_units, other_address_cols]
-                == after_units.loc[mask_moved_units, other_address_cols]
+                before_units.loc[mask_moved_units, address_cols]
+                == after_units.loc[mask_moved_units, address_cols]
             )
             .all(axis=1)
             .mean()
         ) < 0.001  # TODO: pick a smarter number?
-
         # address details do not change if address_id does not change
         pd.testing.assert_frame_equal(
             before_units[~mask_moved_units], after_units[~mask_moved_units]
         )
 
-        # TODO when puma/state is implemented
-        # # TODO Check that *most* are in a new state when we add all locations
-        # # Some movers are in a new state.
-        # # assert any(before.loc[mask_movers, "state"] != after.loc[mask_movers, "state"])
-
-        # # Most movers are in a different puma
-        # assert (
-        #     before.loc[mask_movers, "puma"] != after.loc[mask_movers, "puma"]
-        # ).sum() / mask_movers.sum() >= 0.95
+        # TODO Check that *most* are in a new state when we add all locations
+        # Some movers are in a new state.
+        # assert any(before.loc[mask_movers, "state"] != after.loc[mask_movers, "state"])
```

Comment:
Does this need to still be commented?

Reply:
Yeah, I'm waiting to add all 50+ locations

Reply:
My interpretation of the above comment was that once we had 50, we would test that _most_ move -- but with only 2, it seems like we should be able to test that _some_ move? That said, some of us still regularly run the sim with Florida only. So maybe we should uncomment and explicitly gate on the location in the config?

Reply:
Oh, I see - I just forgot to comment out the "some" movers check. Will fix and keep the TODO to change it later.

I'm going to keep the test as is b/c I'm pretty sure once this gets merged there's no reason to use florida